src/main/java/net/shadowmage/ancientwarfare/structure/block/BlockBrazierEmber.java

```diff
@@ -0,0 +1,230 @@
+import net.minecraft.block.properties.PropertyBool;
+import net.minecraft.block.state.BlockFaceShape;
+import net.minecraft.block.state.BlockStateContainer;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.client.renderer.block.model.ModelResourceLocation;
+import net.minecraft.client.renderer.block.statemap.StateMapperBase;
+import net.minecraft.creativetab.CreativeTabs;
+import net.minecraft.entity.EntityLivingBase;
+import net.minecraft.entity.player.EntityPlayer;
+import net.minecraft.init.Blocks;
+import net.minecraft.init.Items;
+import net.minecraft.init.SoundEvents;
+import net.minecraft.item.Item;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.*;
+import net.minecraft.util.math.AxisAlignedBB;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.RayTraceResult;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.World;
+import net.minecraftforge.client.model.ModelLoader;
+import net.minecraftforge.fml.relauncher.Side;
+import net.minecraftforge.fml.relauncher.SideOnly;
+import net.shadowmage.ancientwarfare.core.AncientWarfareCore;
+import net.shadowmage.ancientwarfare.core.util.NBTBuilder;
+
+import javax.annotation.Nullable;
+import java.util.Random;
+
+public class BlockBrazierEmber extends BlockBaseStructure {
+	public static final String LIT_TAG = "lit";
+	private static final PropertyBool LIT = PropertyBool.create(LIT_TAG);
+
+	private static final AxisAlignedBB AABB = new AxisAlignedBB(0D, 0D, 0D, 1D, 0.88D, 1D);
+
+	public BlockBrazierEmber() {
+		super(Material.ROCK, "brazier_ember");
+		setHardness(2);
+		setTickRandomly(true);
+	}
+
+	@Override
+	protected BlockStateContainer createBlockState() {
+		return new BlockStateContainer(this, LIT);
+	}
+
+	@Override
+	public void getSubBlocks(CreativeTabs itemIn, NonNullList<ItemStack> items) {
+		Item item = Item.getItemFromBlock(this);
+			ItemStack stack = new ItemStack(item);
+			stack.setTagCompound(new NBTBuilder().setBoolean(LIT_TAG, false).build());
+			items.add(stack);
+			stack = new ItemStack(item);
+			stack.setTagCompound(new NBTBuilder().setBoolean(LIT_TAG, true).build());
+			items.add(stack);
+	}
+
+	@Override
+	public IBlockState getStateFromMeta(int meta) {
+		return getDefaultState().withProperty(LIT, (meta >> 3) == 1);
+	}
+
+	@Override
+	public int getMetaFromState(IBlockState state) {
+		return (state.getValue(LIT) ? 1 << 3 : 0);
+	}
+
+	@Override
+	public boolean isFullCube(IBlockState state) {
+		return false;
+	}
+
+	@Override
+	public boolean isOpaqueCube(IBlockState state) {
+		return false;
+	}
+
+	@Override
+	public boolean isNormalCube(IBlockState state) {
+		return false;
+	}
+
+	@Override
+	public AxisAlignedBB getBoundingBox(IBlockState state, IBlockAccess source, BlockPos pos) { return AABB; }
+
+	@Nullable
+	@Override
+	public AxisAlignedBB getCollisionBoundingBox(IBlockState blockState, IBlockAccess worldIn, BlockPos pos) {
```

Comment:
there's no need for the override to getCollisionBoundingBox if it returns the same bounding box as getBoundingBox because in parent Block class getCollisionBoundingBox  actually calls getBoundingBox.